File: 1365. How Many Numbers Are Smaller Than the Current Number/1365. How Many Numbers Are Smaller Than the Current Number.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var smallerNumbersThanCurrent = function(nums) {
    var len=nums.length
    var result=[]
    for(let i=0; i<len;i++){
        var count=0
        for(let j=0; j<len; j++){
            if(i!=j && nums[i]>nums[j]){
                count+=1
            }
        }
        result.push(count)
    }
    return result    
};

File: 1365. How Many Numbers Are Smaller Than the Current Number/1365. How Many Numbers Are Smaller Than the Current Number.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* smallerNumbersThanCurrent(int* nums, int numsSize, int* returnSize) {
    int* result = (int*) malloc(numsSize* sizeof(int));
    int count;
    for(int i=0; i<numsSize; i++){
        count=0;
        for(int j=0; j<numsSize; j++){
            if(i!=j && nums[i]>nums[j]){
                count++;
            }
        }
        result[i]=count;
    }
    *returnSize = numsSize;
    return result;    
}
